### PR TITLE
Feature/quote

### PR DESCRIPTION
**Pull request by EranM6**
**Related issue(s):** #85

## Description

**Notes:**
Missing image, still waiting for the Image cmponent.

## Checklist

Design:
  * [ ] Approved by designer

Usability:
  * [x] Developer
  * [ ] Project manager

Tests
  
  * [x] UI snapshot testing

### CODE REVIEW

File: packages/components/htz-components/src/components/Quote/Quote.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+   */
+  credit: PropTypes.string,
+  /**
+   * Quotes position, **only** in Magazine article (mid-left, mid-right, ect).
+   */
+  position: PropTypes.string,
+  /**
+   * After which paragraph should the Quote be displayed (for Magazine article).
+   */
+  afterParagraph: PropTypes.oneOfType([
```
Comment (TxHawks) [inline packages/components/htz-components/src/components/Quote/Quote.js]: I'm not sure I understand the purpose of this prop... Isn't the position controlled by the parent component? Also can't see how it is used in the code

File: packages/components/htz-components/src/components/Quote/Quote.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  marginBottom: '1rem',
+});
+const TopBorder = createComponent(topBorderStyle, 'span');
+
+const getStyleObj = (theme, quoteType) => {
+  switch (quoteType) {
+    case 'image' :
+      return {
+        ':before': {
+          content: "'\"'",
```
Comment (TxHawks) [inline packages/components/htz-components/src/components/Quote/Quote.js]: ?

File: packages/components/htz-components/src/components/Quote/Quote.js [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+const topBorderStyle = ({ theme, }) => ({
+  borderTopWidth: '1rem',
+  borderTopColor: theme.color('primary'),
+  borderTopStyle: 'solid',
+  width: '16.5rem',
+  height: '1rem',
+  display: 'inline-block',
+  marginBottom: '1rem',
+});
+const TopBorder = createComponent(topBorderStyle, 'span');
```
Comment (TxHawks) [inline packages/components/htz-components/src/components/Quote/Quote.js]: Not a big deal, but I'd use a `::before` pseudo element for the border.

**Comment by TxHawks** [comment]
Eran, can you show this to max and Adi as well (visually I mean, in the styleguide)

File: packages/components/htz-components/src/components/Quote/Quote.js (hunk 1 above)
Comment (EranM6) [reply packages/components/htz-components/src/components/Quote/Quote.js]: not always. in Magazine article its position is set in the element's meta

File: packages/components/htz-components/src/components/Quote/Quote.js (hunk 2 above)
Comment (EranM6) [reply packages/components/htz-components/src/components/Quote/Quote.js]: according to the design, when there's an image, the quoted sentence shuold be surrounded by a quotation marks

File: packages/components/htz-components/src/components/Quote/Quote.js (hunk 3 above)
Comment (EranM6) [reply packages/components/htz-components/src/components/Quote/Quote.js]: It's much easier to control its alignment that way.
And beside that, beause this kind of decoration repeats itself in our sites (at least the current ones), perhaps it shuold be a component of its own ??

File: packages/components/htz-components/src/components/Quote/Quote.js (hunk 1 above)
Comment (TxHawks) [reply packages/components/htz-components/src/components/Quote/Quote.js]: Let's drop this. This isn't something that we'll use in magazine articles either

**Pull request merged**